### PR TITLE
Fix extra space in 3 supplies string

### PR DESCRIPTION
**Pull request by Dimi20cen**
Issue:

Fix: Remove extra space between '3' and 'supplies' in user instruction string

@chimosky @sourabhaa

### CODE REVIEW

File: LemonadeGui.py [hunk 1]
```diff
@@ -135,8 +135,8 @@ def draw_help(self):
         if self.game_mode == 0:
             block = self._blit_to_block([
                 _("""Type in the number you want to buy of each item"""),
-                _("""(be careful: you will need all 3
-                    supplies to make the lemonade)""")])
+                _("""(be careful: you will need all 3 """
+                  """supplies to make the lemonade)""")])
```
Comment (chimosky) [inline LemonadeGui.py]: This is supposed to be one argument and the separate quotes make it difficult to see that.

**Review by chimosky — CHANGES_REQUESTED** [review]

File: LemonadeGui.py (hunk 1 above)
Comment (Dimi20cen) [reply LemonadeGui.py]: The way it was originally it created an issue. To solve said issue another alternative would be to keep the whole string in one line and get a flake8 warning, which I believe is the best approach. You agree?

Comment (chimosky) [reply LemonadeGui.py]: The quotes tell me it's a multi-line comment so let's make it that.

Comment (Dimi20cen) [reply LemonadeGui.py]: Not sure what you mean

Comment (chimosky) [reply LemonadeGui.py]:
```python3
_(
"""
(be careful: you will need all 3 supplies to make the lemonade)
"""
)])
```

Comment (Dimi20cen) [reply LemonadeGui.py]: Ah got it thanks, fixed

File: LemonadeGui.py [hunk 2]
```diff
@@ -135,8 +135,11 @@ def draw_help(self):
         if self.game_mode == 0:
             block = self._blit_to_block([
                 _("""Type in the number you want to buy of each item"""),
-                _("""(be careful: you will need all 3 """
-                  """supplies to make the lemonade)""")])
+                _(
+                    """
```
Comment (chimosky) [inline LemonadeGui.py]: Wrong indentation.

**Review by chimosky — CHANGES_REQUESTED** [review]

File: LemonadeGui.py (hunk 2 above)
Comment (Dimi20cen) [reply LemonadeGui.py]: This causes Flake8(E122)
```
                _(
                """
                (be careful: you will need all 3 supplies to make the lemonade)
                """
                )])
````

Comment (chimosky) [reply LemonadeGui.py]: Then fix it.

Comment (Dimi20cen) [reply LemonadeGui.py]: The following causes Flake8(E122)
```
_(
"""
(be careful: you will need all 3 supplies to make the lemonade)
"""
)])
```

That's why I intended it to look like this so it doesn't cause a Flake8(E122)
```
 _(
    """
(be careful: you will need all 3 supplies to make the lemonade)
"""
)])
```

Do you suggest a different approach?

Comment (chimosky) [reply LemonadeGui.py]: This seems to fix it;
```python3
         if self.game_mode == 0:
             block = self._blit_to_block([
                 _("""Type in the number you want to buy of each item"""),
                 _("""be careful: you will need all 3 supplies to make \
                 the lemonade""")
                 ])
```

Comment (Dimi20cen) [reply LemonadeGui.py]: Fixed

**Pull request merged**